guides/api-query-language-velo.md

```diff
@@ -0,0 +1,267 @@
+# API Query Language 
+
+The query language described in this article
+is implemented partially or in full by certain Wix Velo APIs that support query 
+capabilities.
+
+You may see some similarities between the Wix API Query Language
+and MongoQL. This is because the style of the Wix API Query Language
+is heavily influenced by MongoQL.
+
+## Velo query techniques
+
+2 types of query techniques are available in Velo. See your specific API to check which technique the query function supports. 
+    
+* **Without query builders**: Call query functions to retrieve a list of items using the API Query Language described in this article. Pass an object defining the query as a parameter to the query function.
+
+* **With query builders**: Call query functions that build a query to retrieve a list of items. You can recognize these query functions because they have associated `<item>QueryBuilder` and `<item>QueryResult` class objects. These query functions do not use the API query language syntax described here. 
+
+> **Note**: Querying with query builders is the standard Velo querying technique.
+
+
+## API query language syntax
+
+The object that you pass to the query function can consist of 5 optional parts:
+
+* [`filter`](#the-filter-object):
+  Which results to return.
+* [`sort`](#the-sort-array):
+  In what order.
+* [`paging`](#the-paging-object):
+  Return only some of the matched entities.
+* [`fields`](#the-fields-array):
+  Field projection. Returns only part of each entity.
+* [`fieldsets`](#the-fieldsets-array):
+  Predefined, named sets of fields with common use.
+  This is a shorthand provided by individual APIs.
+
+> **Note**: Usually the query definition is contained in a `query` object. In some cases, the `query` object might be contained inside an `options` object. In other cases, the query's parts might be defined directly inside an `options` object (without a `query` object). See your specific API for information on how to define the object you need to pass to the query function. 
```

Comment:
> In other cases, the query's parts might be defined directly inside an `options` object (without a `query` object). 

"the query's parts" is a little confusing here. Maybe something like this?

```md
In other cases, the query's properties might be defined directly inside an `options` object (without being wrapped in the `query` object). 
```

Or not. Maybe it's just putting lipstick on a pig 🤷

Reply:
No, it's not lipstick! And I'm the pig! 
I also did not like the use of the word "part" but that is what is used in the REST article so I was going to stick with it. But I like your suggestion.  Correcting.